packages/devops/scripts/deployment/buildDeployablePackages.sh

```diff
@@ -12,7 +12,7 @@ PACKAGES=$(${TUPAIA_DIR}/scripts/bash/getDeployablePackages.sh)
 
 # Install external dependencies and build internal dependencies
 cd ${TUPAIA_DIR}
-yarn install
+yarn install --non-interactive --frozen-lockfile
```

Comment:
This matches what we do during codeship builds: don't update dependency versions within their accepted range or try to consolidate versions, or do any other fancy `yarn` stuff...just accept the current version of the lockfile and install the dependencies recorded there.